irohad/main/application.cpp

```diff
@@ -309,12 +313,12 @@ void Irohad::initOrderingGate() {
       shared_model::validation::DefaultProposalValidator>>();
 
   const uint64_t kCounter = 0, kMaxLocalCounter = 2;
```

Comment:
Out of this PR, but the current name is not informative
```suggestion
   const uint64_t kCounter = 0, kDelayStartRound = 2;
```

Reply:
For me personally, the code with these counters does not look valid.
I would not like to change this now. This will be discussed

Reply:
okay